react/components/PriceRange.js

```diff
@@ -23,7 +23,14 @@ const PriceRange = ({ title, facets, intl, priceRange }) => {
     }
 
     navigateTimeoutId.current = setTimeout(() => {
-      setQuery({ priceRange: `${left} TO ${right}`, page: undefined })
+      const urlParams = new URLSearchParams(window.location.search)
```

Comment:
Have you tested in Safari and checked the support of this? Or if we poly fill?

Reply:
Seems to work fine on iOS

Reply:
(Talked to Breno in-person, this is just to keep a record) The problem with using `URLSearchParams` is regarding SSR, which would not be a problem here